website/server/controllers/top-level/dataexport.js

```diff
@@ -25,12 +25,27 @@ const BASE_URL = nconf.get('BASE_URL');
 let api = {};
 
 /**
+ * @apiDefine authError
+ * @apiError NotAuthorized There was an error in authorizing the User ID / API Token pair. Check the <code>message</code> for more information.
+ */
+
+/**
+ * @apiDefine pathParam Path Parameters
```

Comment:
This is defined here, so you can remove it: https://github.com/HabitRPG/habitica/commit/afb5b473a37325a82151bf48b0c7d54b4979e6eb